go.mod

```diff
@@ -56,7 +56,7 @@ require (
 	go.opentelemetry.io/otel/sdk/metric v1.31.0
 	go.opentelemetry.io/otel/trace v1.31.0
 	go.temporal.io/api v1.41.1-0.20241106230151-fd920b39031f
-	go.temporal.io/sdk v1.29.2-0.20241008230001-c82a8ac11cc6
+	go.temporal.io/sdk v1.29.2-0.20241024205028-5505d0467f79
```

Comment:
You should be able to upgrade to the latest stable SDK: https://github.com/temporalio/sdk-go/blob/1f91c8315202b111287c310c197e8d323cbe70b4/go.mod#L12

Reply:
Done